engine/src/main/java/org/terasology/logic/console/commands/CoreCommands.java

```diff
@@ -517,13 +537,87 @@ public String spawnBlock(@Sender EntityRef sender, @CommandParam("blockName") St
         return "Spawned block.";
     }
 
+    @Command(shortDescription = "You can indicate which block to drop with the first parameter and how many with the second", helpText = "the particular block can be seleted which " +
+            "the player wants to drop"
+                , runOnServer = true, requiredPermission = PermissionManager.CHEAT_PERMISSION)
+    public String bulkDrop(@Sender EntityRef sender, @CommandParam("blockName") String blockName, @CommandParam("value") int value) {
+
+        //This is a loop which gives the particular amount of block the player wants to spawn
+        ClientComponent clientComponent = sender.getComponent(ClientComponent.class);
+        LocationComponent characterLocation = clientComponent.character.getComponent(LocationComponent.class);
+
+        Vector3f spawnPos = characterLocation.getWorldPosition();
+        Vector3f offset = characterLocation.getWorldDirection();
+
+        offset.scale(3);
+        spawnPos.add(5,10,0);
+        BlockFamily block = blockManager.getBlockFamily(blockName);
+        if (block == null)
+        {
```

Comment:
Are you sure you've got Checkstyle set up within IntelliJ? Braces like this on the next line is one of the very common violations it catches. Starting braces should be on the same line as the statement that begins the new block :)

Also, you can make the return more informative by including the actual block name in the message